content/docs/jsx-in-depth.md

```diff
@@ -265,30 +265,30 @@ const App = () => {
 };
 ```
 
-In the example above, the `kind` prop is safely consumed and *is not* passed on to the `<button>` element in the DOM.
-All other props are passed via the `...other` object making this component really flexible. You can see that it passes an `onClick` and `children` props.
+Pada contoh di atas, *jenis* (`kind`) *prop* tersebut secara aman digunakan dan *tidak* dioper kepada elemen `<button>` dalam DOM.
+Semua *prop* lainnya dioper melalui objek `...other` membuat komponen ini begitu fleksibel. Anda bisa lihat bahwa komponen tersebut mengoper sebuah *prop* `onClick` dan *prop* `children`.
 
-Spread attributes can be useful but they also make it easy to pass unnecessary props to components that don't care about them or to pass invalid HTML attributes to the DOM. We recommend using this syntax sparingly.  
+Penyebaran atribut bisa berguna tetapi juga membuatnya mudah mengoper *prop* yang tidak perlu ke komponen-komponen yang tidak memperdulikan *prop* tersebut atau mengoper atribut-atribut HTML yang tidak valid ke DOM. Kami rekomendasikan menggunakan sintaks ini seperlunya.  
 
-## Children in JSX {#children-in-jsx}
+## Turunan (*Children*) dalam JSX {#children-in-jsx}
```

Comment:
```suggestion
## *Children* dalam JSX {#children-in-jsx}
```

Let's keep all of these untranslated (*children*)

Reply:
>## *Children* dalam JSX {#children-in-jsx}

Correction applied